src/main/haskell/kore/src/Kore/AST/Common.hs

```diff
@@ -80,18 +75,15 @@ prettyPrintAstLocation :: AstLocation -> String
 prettyPrintAstLocation AstLocationNone = "<unknown location>"
 prettyPrintAstLocation AstLocationImplicit = "<implicitly defined entity>"
 prettyPrintAstLocation AstLocationGeneratedVariable =
-    "<variable generated internally>"
+  "<variable generated internally>"
 prettyPrintAstLocation AstLocationTest = "<test data>"
-prettyPrintAstLocation
-    (AstLocationFile FileLocation
-        { fileName = name
-        , line = line'
-        , column = column'
-        }
-    )
-    = name ++ " " ++ show line' ++ ":" ++ show column'
+prettyPrintAstLocation (AstLocationFile FileLocation { fileName = name
```

Comment:
Please do not reformat code, especially not in a such bad way. Please follow this style guide:
https://docs.google.com/document/d/1h_rNcOzZQi0pCKMCdtmUm-MDKklY33LujpMzCEuQPCQ/edit?ts=5ada4619#

This comment is valid for this entire pull request.